web/templates/foi_requests/publicbody_form.html

```diff
@@ -59,3 +70,24 @@ <h2>Cadastrar órgão público</h2>
   </form>
 </div>
 {% endblock %}
+
+{% block scripts %}
+<script src="{% static 'js/jquery.min.js' %}"></script>
```

Comment:
Ao invés de usar o JQuery, usa JavaScript puro, assim a gente evita adicionar ~100 KB à página. O código fica praticamente o mesmo, só usa `element.addEventListener('change', function() {` e `$('#meu_id')` por `document.getElementById('meu_id')`.

O https://blog.garstasio.com/you-dont-need-jquery/events/#listening-for-events pode ajudar